README.md

```diff
@@ -1,24 +1,213 @@
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Spash Screen**
+- **Sign up and log in pages**
+- **Home page (catagories page)**
+- **Transactions page**
+- **"Add a new category" page**
+- **"Add a new transaction" page**
+- **Unit & Integration Testing with RSpec & Capybara**
+- **Aunthentication with Devise**
+- **Database management with Postgres**
+- **ORM with ActiveRecord**
+- **Deployment**
+- **Project Demo with Loom**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> View Video Demo [here](https://www.loom.com/share/0ff174560ebd4edfa4ee68f389f36836)
+
+> View Live Demo [here](https://fonkwa.onrender.com/) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+ Ruby installed
+ Postgresql installed
+
+ gem install rails
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone https://github.com/Gleennkar/Budget-App-RoR.git
+
+  cd budget-app
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+  rails server
+
+### Run tests
+
+To run tests, run the following command:
+
+  rspec
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+**Odette Gleen**
+
+- GitHub: [@Gleennkar](https://github.com/Gleennkar)
+- Twitter: [@gleeny24](https://twitter.com/twitterhandle)
+- LinkedIn: [@gleeny](https://www.linkedin.com/in/gleeny-nkar-aa3917182)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+>
+
+- [ ] **Add API Endpoints**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Gleennkar/Budget-App-RoR/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please add a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Credit to Microverse for the project specification
+
+Credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for the design guidelines.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](.//https://github.com/Gleennkar/Budget-App-RoR/edit/Dev/LICENSE) licensed.
```

Comment:
- [ ] Good job on making efforts in implementing changes suggested by the previous reviewer 👏 Hower, I still can't find your Licence file when I click on this link. This is because you have not added the license file in your Repo. It should be at the root level of your project/repo, then on the above link, you add the path to that file. That means when you click on the link, it will open the license file in your repo 😃 😉 
You can use the license file you used in your previous projects or if you don't have it, you can use the one found [here](https://github.com/BertrandConxy/GOT-Game-Of-Thoughts/blob/dev/LICENSE)